xup/xup.h

```diff
@@ -154,8 +155,8 @@ struct mod
   int height;
   int bpp;
   int sck_closed;
-  char username[256];
-  char password[256];
+  char username[INFO_CLIENT_MAX_CB_LEN];
+  char password[INFO_CLIENT_MAX_CB_LEN];
```

Comment:
@matt335672 in `xup.h` I made the change. Or is there anything else you are referring to?

Reply:
In `xup/xup.c` at (your) line 1549 and 1553, the code to initialise these two fields is still capping the lengths at 255 rather than `INFO_CLIENT_MAX_CB_LEN-1`. 

I don't think the VNC client needs updating at all - sorry about the noise. I see from the comments above that you were looking in this area with earlier commits which have now been removed. The VNC username/password interface could certainly do with tidying up, but that can be left alone for now.